### ISSUE TITLE
Converted nets predict same wrong outputs ( 600,463)  for every input image

### ISSUE THREAD

**Issue by sagieppel** [clcarwin/convert_torch_to_pytorch#31]
I manage to load the trained weight  (resnext 50 and resnext 101).
In both cases the the nets seem to predict 600 or 463 irregardless of the input image.

this is the code I run after the net is built in the py file:
> resnext_101_32x4d.load_state_dict(torch.load("resnext_101_32x4d.pth"))
> resnext_101_32x4d=resnext_101_32x4d.cuda()
> inp=cv2.imread("TREE.jpg") #Load image of tree
> inp=misc.imresize(inp,(216,216)).astype(np.float32)
> inp=np.expand_dims(inp,axis=0)
> inp=torch.autograd.Variable(torch.from_numpy(inp).cuda(),requires_grad=False)
> inp=inp.transpose(1,3).transpose(2,3)
> x=inp
> #x=(inp-125)/64 # I try with and without this part the result still the same
> x=resnext_101_32x4d(x)
> x=F.softmax(x)
> print(np.argmax(np.array(x.data)))
output is 463

**Comment by sagieppel** [comment]
Ok for some reason adding  model.eval() after loading weights solve this problem.


> resnext_101_32x4d.load_state_dict(torch.load("resnext_101_32x4d.pth"))
> resnext_101_32x4d.eval() #adding this solved the problem
> resnext_101_32x4d=resnext_101_32x4d.cuda()
> inp=cv2.imread("DOG.jpg")
> inp=misc.imresize(inp,(216,216)).astype(np.float32)
> inp=np.expand_dims(inp,axis=0)
> inp=torch.autograd.Variable(torch.from_numpy(inp).cuda(),requires_grad=False)
> inp=inp.transpose(1,3).transpose(2,3)
> x=inp
> x=(inp-125)/64 
> x=resnext_101_32x4d(x)
> x=F.softmax(x)
> print(np.argmax(np.array(x.data)))
